README.md

```diff
@@ -22,17 +24,24 @@ hubot environment add staging to foobar
 
 hubot environment add production to foobar
 #=> environment production added to foobar
+```
 
+# Release
 
+```
 # Ask if you can release that application to that environment
 hubot can I release foobar to staging?
 #=> yes, foobar is releasable to staging
 
-
 # Release foobar/master to staging
 hubot release foobar/master to staging
-#=> user is now releasing hubot\/master to staging
```

Comment:
Do we no longer support forever locking outside of using a large number in minutes?